static/js/src/advantage/react/components/Subscriptions/Content/Content.tsx

```diff
@@ -38,6 +38,14 @@ const Content = () => {
     }
   }, [selectedId, setSelectedId, allSubscriptions, isLoading]);
 
+  if (isLoading) {
+    return (
+      <Card className="u-no-margin--bottom" data-test="initial-load">
```

Comment:
@huwshimi @hatched Should we start using `data-testid` instead of `data-test` regardless of whether a test is written with enzyme or testing-library?

`data-testid` is the convention used for testing-library `getByTestId` matcher.

Arguably it's a better (more explicit) name, plus it's good for consistency and will make switching easier.

https://testing-library.com/docs/queries/bytestid/

Reply:
This seams like a reasonable request. I'd just ask that we convert the entire file at once so that we don't have some `test` and some `testid` in the same file.

Reply:
Yep, something to introduce across the entire ua-table section of the site, but this is probably not the right time to do that as we're wrapping things up.